### ISSUE TITLE
Feedback on user manual

### ISSUE THREAD

**Issue by riban-bw** [polluxsynth/audio-plugins#8]
I am reading the preliminary (draft) user manual from [here](https://butoba.net/downloads/mimid-manual.pdf). Here are some observations:

- Introduction says it is a plugin but not what technologies. It _may_ be advantageous to list them, e.g. LV2. This may clarify what is meant by "host" later on in the introduction.
- "Memory" is referenced. Is this "Memory Moog"?
- Should "Mannerheim" be "Oberheim"?
- In section "Architecture" the two envelope generators are mentioned. Is it worth metioning thier type here, e.g. " two ADSSR envelope generators...". Maybe not necessary but I found myself eager to know. But I do see that reading the subsequent bullet list does resolve this desire for knowledge!
- "...y, how many voices are played in unison in the Mono modes." I am eager to understand what this means - I will read on...
- "Single Trig" - the name of the modes is confusing. "N Off" relates to activity for note-on events only. I wonder if there is a more logical label?
- "Assign Order" - I am going to have to try this to better understand, e.g. is "lowest voice" the left most on a keyboard or an internal logical assignment?
- A general comment about key assignment: This is a very flexible implementation that is likely to confuse some users. It is the kind of thing that one understands once then forgets later. It may benefit from some example workflows. I wouldn't put such examples here but instead maybe add an appendix that shows various examples of the various parameters.
- ~Modulator Controller may be "Modw" which one assumes is MIDI CC 1. In Zynthian we use MIDI mapping (learning) to assign CC to controllers. CC are filtered out from reaching the engines by default. I assume that setting Controller to Modw is likely to have little effect other than to avoid vel & touch from having an effect.~ Modulation Wheel is assigned in the first page (controllers).
- "Sync" says, "The Clock position synchronizes the LFO to the currently playing MIDI temp [not yet implemented]." Should that be "...tempo..."?
- Oscillator detune is up to +1 semitone. Would this benefit from being +/- 1 semitone? (More a feature request than a documentation observation!)
- Oscillator pitch [0..72] - what is the reference? Is 57 = 440Hz?
- Oscillator Wave – "Waveform. Self explanatory...". You then go on to explain! I would suggest removing "Self explanatory". Also I would suggest "Sawtooth (Saw)" for consistency.
- Waveform Reset - I suggest highlighting the modes in bold, e.g. FreeRun, KeySync.
- This may seem petty but seeing "Oscillator 3" suddenly appear in the Mixer section is a suprise. Of course it is quickly explained that this is the sub oscillator. I think it may be more user-friendly to refer to OSC 3 as Sub Osc (or just "SUB") throughout.
- Filter Configuration: Linear Env - This is a toggle that changes the envelope shape. I understand the idea of a switch that enables "Linear" but you also have the option / opportunity to have it toggle between values "Log" and "Lin", maybe with the title "Filter Env Shape".
- VCA HPFfreq talks of, "The nonzero lowest value...". There is no description of what a zero value does. One might infer that it bypasses the HPF.
- VCAdrive has a type: "...can add som ‘zing’...". "Some" is missing its 'e'.
- VCAdrive mentions distorition and flattening but not clipping. I would suggest fitting in the word "clipping" somewhere so that someone searching the document for this common term will find it. Don't change the paragraph to add more - just somehow fit in the word, e.g. by replacing another. Leave "distorition" in there (at least once) as this is also a common term that someone might search for. Maybe replace one of the distoritons with clipping.
- "HPFfreq", "VCAdrive", "EnvMode" have no spaces. I know you are refactoring the control names / symbols. It may be advantageous to add spaces. Some hosts (including Zynthian) split long names at spaces which can improve display. Zynthian changes the font size if it needs to squeeze in longer names which can look odd.
- VCA EnvMode has a typo: "...with employs a linear envelope curve but exponential VCA response." I think "with" should be "which". Also, " an attempt att achieving". "att" should be "at".
- Filter Envelope seems to have a typo: "Additionally, a linear can be selected...". Is the 'a' superfluous?
- I can't find where to set this linear mode. It doesn't seem to be in the Filter Envelope page. Can you reference the control's location?
- Loudness Envelope "...Release parameter being on a separate Env Release page common to both envelopes." Users may infer there is a single release control that is common to both envelopes. (Of course they will soon see this to be false.) I think you can lose "common to both envelopes" which makes the docs accurate and does not lose any context.
- Controller Sens. Velocity Scale seems to have a typo: "Not that all velocity values are still eminently reachable..." should probably be "Note" rather than "Not".
- AfterTouch Scale – Does this work for both channel aftertouch and polyphonic aftertouch? I think it should be explicitly specified.
- FilterEnv Velocity - I would reword a bit to make it easier to read / understand: ": When medium velocity (MIDI value 64) is applied...". Moving the word "velocity" makes it clear what is meant. I had to pause for many seconds and re-read a few times before I understood the current wording.
- Spread - The summary is quite complete but maybe a bit confusing or at least it left me with a question. Is this modelling the drift that analogue synths have due to temperature changes, component tolerances, etc.? If so it might be worth adding such a description to the summary. A single sentence would explain this conscisely which could precede the subsequent more detailed descripton. Actually, re-reading the summary I see you do mention component tollerances and maybe it does not drift with time so I may have misunderstood its purpose.

I must congratulate you on an excellent manual. It is easy to read with an almost conversational feeling. It may benefit from some diagrams but that will take extra time / effort that could delay more important stuff like fixing bugs and implementing feature requests. Of course you can ignore my comments, some of which may be biased but I reported (almost) everthing that occured to me as I read. I wanted to be as useful (and critical) as I could.

**Comment by polluxsynth** [comment]
Thanks for your most excellent read-through, with the additional benefit of reading it for the first time, which catches a lot of inconsistencies that subsequent reads won't find.

I laughed at the "Mannheim" point - before creating the pdf I realized I hadn't done a spell check, so I did one quickly, trying at best to navigate between the 'Correct' and 'Add to dictionary' buttons; clearly I clicked the wrong one this time.

Regarding aftertouch, currently, MiMi-d only responds to channel aftertouch (which is something that should be noted in the manual), but it would be a good idea to have it respond to polyphonic aftertouch as well.

Regarding CC response, I believe that Zynthian somewhere has a default mapping for certain controllers, by default is CC 1 and 64, so that these are passed through to the plugin, and also referenced on the first parameter page. Certainly both mod wheel and sustain pedal work out of the box.

Regarding 'Single trig', perhaps it would be better to reverse the name of the parameter, so it is called 'Multi trig', which might make the values more self evident.

I like the 'Envelope Shape' with the values 'Log' / 'Lin', rather than a 'Linear on/off' mode switch.

Good point about what the Spread functions actually model. It is true that there is no long time variation, although the actual variation will be different whenever the plugin is reloaded.

Most of the comments I think are very helpful. Thanks! I'll try to get them into the next edition of the manual (and also where applicable the relevant changes in the parameter definitions).

**Comment by polluxsynth** [comment]
I'm considering changing Single trig to Multi trig, with the values "All", "Note On", "None". The underlying functionality is identical, but might be easier to understand this way. What to do you think?

**Comment by riban-bw** [comment]
> I'm considering changing Single trig to Multi trig, with the values "All", "Note On", "None". The underlying functionality is identical, but might be easier to understand this way. What to do you think?

I think the control should be called "Env Retrig" to indicate it controls the retriggering of envelope curves. It isn't the initial trigger and isn't really related to single/multiple things. "Env" shows it relates to envelopes and "Retrig" (or "Retrigger") shows it relates to how a key acts after a trigger has already occured.

The values, "All", "Note ON", "None" make perfect sense.

**Comment by ToFFmashines** [comment]
Hi, I offer help with creating diagram images for the manual. All i need is a hand drawn idea of the image which can be photographed or scanned. I would then take care of the redrawing into svg or other required format.
ToFF

**Comment by polluxsynth** [comment]
Thanks, that would be very helpful!

I'll see what I can put together within the next couple of days.

The main need I think is a block diagram.

**Comment by polluxsynth** [comment]
Most issues should now be fixed, and the manual is now included with the source in the plugins/mimid/Doc directory.
